wri/earth-dashboard
# Back office: dynamic text widget

![imagen](https://user-images.githubusercontent.com/545342/92569284-b420b700-f280-11ea-9dd1-481363b08eb0.png)

## Overview
This PR adds support for dynamic text widgets in the back office.

## Testing instructions
Check this URL as an example http://localhost:9000/admin/data/widgets/8383ea30-1166-415d-879f-51e0e8e353c6/edit?dataset=47dc1a1b-2c91-4d69-b04e-ea8c4561e2b5

## [Pivotal task](https://www.pivotaltracker.com/story/show/174693246)
